locks/oldspinlock.h

```diff
@@ -71,10 +71,14 @@ extern "C" size_t MyInterlockedExchange (size_t * oldval,
 #define _MM_PAUSE  {__asm{_emit 0xf3};__asm {_emit 0x90}}
 #include <windows.h>
 
-#elif defined(__GNUC__)
+#elif defined(__GNUC__) && (defined(__i386__) || defined(__x86_64__))
```

Comment:
As mentioned in a comment, this file is only present for compatibility with very old builds.